lib/ansible/modules/cloud/ovirt/ovirt_vm.py

```diff
@@ -2075,11 +2075,17 @@ def main():
         auth = module.params.pop('auth')
         connection = create_connection(auth)
         vms_service = connection.system_service().vms_service()
+
+        if not module.params['wait'] and state=='running':
```

Comment:
If I understand correctly now it will wait always, but we want only to wait to be created , not for running.